crypto/src/mls/external_commit.rs

```diff
@@ -76,21 +75,19 @@ impl MlsCentral {
     /// bad can happen if you forget to except some storage space wasted.
     ///
     /// # Arguments
-    /// * `group_info` - a GroupInfo wrapped in a MLS message. it can be obtained by deserializing a TLS
-    /// serialized `GroupInfo` object
+    /// * `group_info` - a GroupInfo wrapped in a MLS message. it can be obtained by deserializing a TLS serialized `GroupInfo` object
     /// * `custom_cfg` - configuration of the MLS conversation fetched from the Delivery Service
     /// * `credential_type` - kind of [openmls::prelude::Credential] to use for joining this group.
-    /// If [MlsCredentialType::Basic] is chosen and no Credential has been created yet for it,
-    /// a new one will be generated. When [MlsCredentialType::X509] is chosen, it fails when no
-    /// [openmls::prelude::Credential] has been created for the given Ciphersuite.
+    ///   If [MlsCredentialType::Basic] is chosen and no Credential has been created yet for it,
+    ///   a new one will be generated. When [MlsCredentialType::X509] is chosen, it fails when no
+    ///   [openmls::prelude::Credential] has been created for the given Ciphersuite.
```

Comment:
Were these formatting changes made accidentally?

Reply:
Was that `cargo fmt` or something else?

Reply:
I don't remember if it was clippy or fmt